metta_amr/metta_space/pattern_parser.py

```diff
@@ -42,6 +43,13 @@ def _process_triples(self, triples):
             # else:
             self.amr_space.add_triple(triple)
 
+    def load_templates_from_files(self, filenames):
+        if not isinstance(filenames, list):
+            filenames = [filenames]
+        for filename in filenames:
+            self.load_templates_from_file(filename)
+        self.amr_space.index_amrsets()
```

Comment:
I've add this function because we need to call index_amrsets() after all  templates were loaded